action.yml

```diff
@@ -16,6 +16,12 @@ inputs:
     description: "Framework for Tests"
     required: false
     default: "pytest"
+  github_token:
+    description: "Github token"
+    required: true
+    default: "dummy-token"
```

Comment:
Github Token should not have a default similar to ecr_repository

Reply:
done